pep-0675.rst

```diff
@@ -747,13 +737,67 @@ others may consider it in the future, such extensions are out of scope
 for this PEP.
 
 
+Rejected Names
+--------------
+
+We considered a variety of names for the literal string type and
+solicited ideas on `typing-sig
+<https://mail.python.org/archives/list/typing-sig@python.org/thread/VB74EHNM4RODDFM64NEEEBJQVAUAWIAW/>`_. Some
+notable alternatives were:
```

Comment:
```suggestion
<https://mail.python.org/archives/list/typing-sig@python.org/thread/VB74EHNM4RODDFM64NEEEBJQVAUAWIAW/>`_.
Some notable alternatives were:
```

Formatting nitpick

Reply:
Can we add a lint rule for this? I like the other RST checks as part of the pre-commit tests. I don't know what is wrong here. (I just use `fill-paragraph` in Emacs.)

Reply:
Its just a formatting nitpick; doesn't actually affect anything. For whatever reason `fill-paragraph` is not adding the break before `Some` despite it being at the end of a sentence and it being far over the maximum column width, either because it is getting confused about RST syntax or it doesn't have two spaces after periods. Allowing One Line Per Sentence fur future PEPs instead of just old-fashioned hard breaks would fix all of these issues and much more, and make reviews much easier and diffs much cleaner, but I've yet to formally propose it since I've been focused on more important things first.